wiki/mapping/extended-mapping.md

```diff
@@ -10,9 +10,69 @@ _A quick summary of Extended Mapping_
 This is a stub page. Content to come soon!
 :::
 
-## Multi-Lane Mapping
-## Precision Placement
-## Precision Rotation
-## Wall Mapping
-## Bomb Mapping
-## Extended Characteristics
+## Mapping Extensions
+
+### Set-Up
+
+**Install Mapping Extensions with Beat Saber Installed**
+
+Note: This section assumes you have not pointed Mediocre Map Assistant 2 to a different location for your WIP maps. Skip ahead if you don’t have the game or want to store your WIP maps elsewhere.
+
+1. Start Beat Saber and then close out after the game finishes loading. This is to ensure the game is ready for mods to be installed.
+2. Open [ModAssistant](https://github.com/Assistant/ModAssistant/releases/latest)
+3. Make sure the version selected in the dropdown is compatible with the Beat Saber version installed on your computer.
+4. Check the checkbox for MappingExtensions. (You can select additional mods to install as well.)
+5. Click Install and Update and make sure there are no errors.
+6. Open Beat Saber again to initialize the mods you have installed then quit the game.
+
+To check if it worked navigate to your Beat Saber install and make sure that a MappingExtensions.dll file is present.
+
+Oculus: `\Oculus Apps\Software\hyperbolic-magnetism-beat-saber\Plugins`
```

Comment:
There is a page in the wiki FAQ that explains where the install folder is located.
To remove redundancy and save space on here, consider replacing this with something like
`Not sure where Beat Saber is installed? See [install folder](`/faq/install-folder.md`)`